### ISSUE TITLE
error when migrate in postgres

### ISSUE THREAD

**Issue by inzem77** [awesto/django-shop#677]
i do ./manage.py migrate
in migration shop.0008_notification_recipient.py happends error 
 
```python
File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/core/management/commands/migrate.py", line 204, in handle
    fake_initial=fake_initial,
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/migrations/executor.py", line 115, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/migrations/executor.py", line 145, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/migrations/executor.py", line 244, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/migrations/migration.py", line 129, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/migrations/operations/fields.py", line 204, in database_forwards
    schema_editor.alter_field(from_model, from_field, to_field)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/backends/base/schema.py", line 495, in alter_field
    old_db_params, new_db_params, strict)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/backends/postgresql/schema.py", line 117, in _alter_field
    new_db_params, strict,
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/backends/base/schema.py", line 578, in _alter_field
    new_default = self.effective_default(new_field)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/backends/base/schema.py", line 221, in effective_default
    default = field.get_db_prep_save(default, self.connection)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/models/fields/__init__.py", line 755, in get_db_prep_save
    prepared=False)
  File "/home/inzem/sites/django_shop/ENV/lib/python3.5/site-packages/django/db/models/fields/__init__.py", line 747, in get_db_prep_value
    value = self.get_prep_value(value)
  File "/home/inzem/sites/django_shop/django-shop/shop/models/fields.py", line 99, in get_prep_value
    raise ValueError("Value must be of type {}".format(self.enum_type))
ValueError: Value must be of type <enum 'ChoiceEnum'>

```

**Comment by jrief** [comment]
not enough information to reproduce this

**Comment by gntjou** [comment]
I get exactly the same error when installing a fresh django-shop tutorial on a freshly installed debian jessie server using mysql as database server. I have tried it numerous times now, but every time the migration stops at shop.0008. I followed the djangoShop pdf document 0.11.2 to the letter and don't get any error nor warning messages up to this one during ./manage.py migrate

  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/core/management/commands/migrate.py", line 204, in handle
    fake_initial=fake_initial,
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/db/migrations/executor.py", line 115, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/db/migrations/executor.py", line 145, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/db/migrations/executor.py", line 244, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/db/migrations/migration.py", line 129, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/db/migrations/operations/fields.py", line 204, in database_forwards
    schema_editor.alter_field(from_model, from_field, to_field)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/db/backends/base/schema.py", line 495, in alter_field
    old_db_params, new_db_params, strict)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/db/backends/base/schema.py", line 578, in _alter_field
    new_default = self.effective_default(new_field)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/db/backends/base/schema.py", line 221, in effective_default
    default = field.get_db_prep_save(default, self.connection)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/db/models/fields/__init__.py", line 755, in get_db_prep_save
    prepared=False)
  File "/home/frank/webshop/faktorie77/lib/python3.4/site-packages/django/db/models/fields/__init__.py", line 747, in get_db_prep_value
    value = self.get_prep_value(value)
  File "/home/frank/webshop/django-shop/shop/models/fields.py", line 98, in get_prep_value
    raise ValueError("Value must be of type {}".format(self.enum_type))
ValueError: Value must be of type <enum 'ChoiceEnum'>

**Comment by jrief** [comment]
closed via #678

Thanks for reporting

**Comment by jrief** [comment]
ups, this is related to another function.

**Comment by gntjou** [comment]
I tried fix #678 but it didn't solve the problem. Shop.0008 still give the same error

**Comment by jrief** [comment]
It nor should be fixed. Please retry.
